deployment.yaml

```diff
@@ -15,7 +15,7 @@ spec:
         env: canary
     spec:
       containers:
-        - image: REPLACE_IMAGE
+        - image: DOCKER_IMAGE
```

Comment:
what does DOCKER_IMAGE refer to ? Is it a placeholder ?

Reply:
It is a placeholder replaced in bash script https://github.com/andrewkandzuba/openexchange-webservice-go/blob/ddf4bfd1ff704cb8d520eedbdea97b12be721867/deploy.sh#L6

Reply:
see new comment in README.md (i.e. document this better)

Reply:
Added note about `DOCKER_IMAGE` environment variable into [this section](https://github.com/andrewkandzuba/openexchange-webservice-go/tree/google-cloud-build#12-deploy-to-google-kubernetes-engine-gke)